src/java/com/sarality/app/common/datetime/DateTimeUtils.java

```diff
@@ -0,0 +1,31 @@
+package com.sarality.app.common.datetime;
+
+import hirondelle.date4j.DateTime;
+
+/**
+ * Utility class to perform simple DateTime conversions and comparisons.
+ *
+ * @author abhideep@ (Abhideep Singh)
+ */
+public class DateTimeUtils {
+
+  public static DateTime toDateOnly(DateTime dateTime) {
+    return DateTime.forDateOnly(dateTime.getYear(), dateTime.getMonth(), dateTime.getDay());
+  }
+
+  public static DateTime toTimeOnly(DateTime dateTime) {
+    return DateTime.forTimeOnly(dateTime.getHour(), dateTime.getMinute(), dateTime.getSecond(),
+        dateTime.getNanoseconds());
+  }
+
+  public static boolean isSameTime(DateTime lhs, DateTime rhs) {
```

Comment:
ok. good to go!

On 10-Mar-2015, at 11:14 am, Abhideep Singh <notifications@github.com> wrote:

> In src/java/com/sarality/app/common/datetime/DateTimeUtils.java:
> 
> > + * Utility class to perform simple DateTime conversions and comparisons.
> > + *
> > + * @author abhideep@ (Abhideep Singh)
> > + */
> > +public class DateTimeUtils {
> > +
> > +  public static DateTime toDateOnly(DateTime dateTime) {
> > +    return DateTime.forDateOnly(dateTime.getYear(), dateTime.getMonth(), dateTime.getDay());
> > +  }
> > +
> > +  public static DateTime toTimeOnly(DateTime dateTime) {
> > +    return DateTime.forTimeOnly(dateTime.getHour(), dateTime.getMinute(), dateTime.getSecond(),
> > +        dateTime.getNanoseconds());
> > +  }
> > +
> > +  public static boolean isSameTime(DateTime lhs, DateTime rhs) {
> The reason I didn't use isEqual was because
> 
> 1) There is already a method equals() on DateTime, which unfortunately doesn't do what we want
> 
> 2) We need to indicate that we are comparing only times here and not the entire date and times.
> 
> Can discuss when I come in. Leaving now
> 
> —
> Reply to this email directly or view it on GitHub.
>